src/material/schematics/ng-update/upgrade-rules/hammer-gestures-v9/hammer-gestures-rule.ts

```diff
@@ -183,11 +210,11 @@ export class HammerGesturesRule extends MigrationRule<null> {
     // output could also be from a component having an output named similarly to a known
     // hammerjs event (e.g. "@Output() slide"). The usage is therefore somewhat ambiguous
     // and we want to print a message that developers might be able to remove Hammer manually.
-    if (!this._usedInRuntime && this._usedInTemplate) {
-      this.printInfo(chalk.yellow(
+    if (!hasCustomGestureConfigSetup && !this._usedInRuntime && this._usedInTemplate) {
+      this.printInfo(
           'The HammerJS v9 migration for Angular components migrated the ' +
           'project to keep HammerJS installed, but detected ambiguous usage of HammerJS. Please ' +
-          'manually check if you can remove HammerJS from your application.'));
+          'manually check if you can remove HammerJS from your application.');
```

Comment:
Is there a docs page/section that we can point them to? How will they know if they can remove HammerJS from their app?

Reply:
Good question. I asked in Slack. Let's track that there.